tinker-actions/src/fde_dmv/emt_fde_dmv.md

```diff
@@ -0,0 +1,166 @@
+# Full Disk Encryption (FDE) and Device Mapper Verity (DM-Verity)
+
+This document provides an explanation of the partition scheme used in Full Disk Encryption (FDE) and Device Mapper Verity (DM-Verity), along with the key differences between the two mechanisms.
+
+---
+
+## Requirements
+
+### System Requirements
+
+#### Hardware:
+- A Trusted Platform Module (TPM) device for key sealing and secure boot.
+- A system with a single or multiple hard disk drives (HDDs) or NVMe drives.
+- Minimum Disk Size required: 128GB.

```

Comment:
Here it is mentioned 128GB but if we add the partitions size mentioned in the partition scheme below it will come around ~50GB. Are missing something here?

Reply:
We are calculating the sizes of the rootfs and other partitions based on a128GB HDD. 
we use 100GB for all of rootfs and keep the rest of the disk as lvm partition. this is part of our requirements.